test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -570,77 +914,274 @@ define([
                         });
                     });
                 });
-            });
 
-            describe('job params', () => {
-                const tests = [
-                    {
-                        input: {
-                            job_params: [{ this: 'that' }],
-                        },
-                        regex: new RegExp('this: that'),
-                    },
-                    {
-                        input: {
-                            job_params: [{ tag_two: 'value two', tag_three: 'value three' }],
-                        },
-                        regex: new RegExp('tag_three: value three.*?tag_two: value two', 'ism'),
-                    },
-                ];
+                describe('batch with retries', () => {
+                    const batchJob = JobsData.batchJob,
+                        batchParentJob = batchJob.jobsById[batchJob.batchId];
+                    let allRows;
+                    beforeEach(async function () {
+                        container = document.createElement('div');
+                        // update the child jobs in the batch parent
+                        batchParentJob.child_jobs = Object.keys(batchJob.originalJobs);
+                        // create a table from the original jobs and the batch container
+                        const originalJobsIds = Object.keys(batchJob.originalJobs).concat(
+                            batchJob.batchId
+                        );
+                        const originalJobsData = originalJobsIds.map((jobId) => {
+                            return batchJob.jobsById[jobId];
+                        });
+                        this.jobManager = new JobManager({
+                            model: makeModel(originalJobsData),
+                            bus: Runtime.make().bus(),
+                        });
+                        this.jobStatusTableInstance = await createStartedInstance(container, {
+                            jobManager: this.jobManager,
+                        });
+
+                        this.jobManager.addListener('job-retry-response', originalJobsIds);
+                    });
+
+                    it('sets up a table correctly', function () {
```

Comment:
Make sure that when a table with retried jobs in it is initialised, the correct jobs are shown.